github_pr.py

```diff
@@ -369,8 +398,21 @@ def main():
 Delete a PR
 
     github-pr delete -r dataxu/test_repo -n 17
+
+Check conditional status checks
+    !!!This check only looks at comments AFTER the latest commit, to validate that the
+    code has been most recently peer reviewed.!!!
+
+    github-pr check-condition -r dataxu/dcommand -n 84 --condition-owner-cannot-ship
+        This will check to make sure that the owner can not apply a shipable comment on their own code
+    github-pr check-condition -r dataxu/dcommand -n84 --condition-approved-merger-file-required=<MAINTAINERS FILE>
+        Compares commenter to list from a file, single user per line, and checks to make sure they are a valid merger
+    github-pr check-condition -r dataxu/dcommand -n 84 --condition-approved-mergers-required=ned_flanders,marge_simpson
+        This will check that a comment containing a :shipit: (the default, or other defined comment if merge_comment is set)
+        comes from a user in the provided list passed on the commandline. The "MAINTAINERS file" option above is the preferred
+        convention to use, and this option is primarily for local testing when setting up your CD flow.
```

Comment:
I think we want to emphasize that it is the most recent code that needs
    review, and saying "most recent code" seems to leave less room for
    ambiguity than "most recently peer reviewed.":
    
    currently:

    `to validate that the code has been most recently peer reviewed`

    proposed:

    `to validate that the most recent code pushed has been peer reviewed`

    I think "approved merger" is more consistent with other language
    and the commandline option we're using:

    currently:

    `to make sure they are a valid merger`

    proposed:

    `to make sure they are an approved merger`

    I think "and this option" can be somewhat ambiguous in referring
    to the "MAINTAINERS file" option that we mention in the first part
    of the sentence, we could better distinguish which option we are
    talking about in the latter part:

    currently:

    `and this option is primarily for local testing when setting up your CD flow.`

    proposed:

    `while this ad-hoc list of comma separated names option is primarily for local testing when setting up your CD flow.`